src/components/Modal/AddLiquidity.vue

```diff
@@ -306,17 +307,24 @@ export default {
       if (this.tokenError || this.validationError) {
         return undefined;
       }
-      for (const token of this.pool.tokensList) {
+      for (const token of this.pool.tokens) {
+        const tokenAddress = token.checksum;
+
         if (
           isLocked(
             this.web3.allowances,
-            token,
+            tokenAddress,
             this.web3.dsProxyAddress,
-            this.amounts[token],
-            this.web3.tokenMetadata[token].decimals
+            this.amounts[tokenAddress],
+            this.web3.tokenMetadata[tokenAddress].decimals
           )
         ) {
-          return `${this.$t('unlock')} ${token.symbol} ${this.$t(
+          const displaySymbol =
+            typeof token.symbol === 'undefined'
+              ? shortenAddress(tokenAddress)
```

Comment:
You can get the same method from the mixins like this:
```suggestion
              ? this._shortenAddress(tokenAddress)
```

Reply:
Ok; updated. I don't entirely understand my own fix (why we have two token lists and this one doesn't have the symbol), but it seems to work